### DISCUSSION TITLE
Migrate Kamal Discord to Campfire installation

### DISCUSSION THREAD

**Discussion by nickhammond** [basecamp/kamal#1022] [Ideas]
As I'm waiting for Discord to install yet another update for the 10th time this week, I had the idea of just migrating discussions over to Campfire instead. 

<img width="290" alt="image" src="https://github.com/user-attachments/assets/8784de0b-ec3a-4b90-b9c7-4e153165a585">

Some other things that came to mind would be utilizing bots in Campfire to post new PRs/issues for kamal-site and kamal into a channel within Campfire. Posting those updates could also just be done via Discord of course.

I know there have been a few accounts banned from Discord because of spam but not sure how much additional moderation is happening or being leveraged with Discord at the moment that Campfire doesn't yet support.

**Comment by dhh** [comment]
Campfire isn't currently well-suited for fully public chat rooms because the moderation tools are quite limited. It works great for invitation-style rooms, like what we did for Rails World, but I don't think it'll work for this (yet??).
